package.json

```diff
@@ -24,7 +24,8 @@
     "build": "webpack --mode production",
     "prettier": "prettier --write '**/*.{js,jsx}'",
     "lint": "node_modules/eslint/bin/eslint.js src --fix",
-    "fix-styles": "prettier-stylelint --write 'src/**/*.{css,scss}'"
+    "sass-lint": "sass-lint 'src/**/*.scss' -q -v",
+    "sass-lint:fix": "sass-lint-auto-fix"
```

Comment:
I think it would also be nice to include this in the pre-commit hook.

Reply:
I do need help on it, how would that be implemented, that is adding the command to the pre-commit-hook @igbominadeveloper

Reply:
Just look at the pre-commit hook in husky of the package.json and add the sass-lint script to it

Reply:
```
"*.scss":[
      "npm run sass-lint:fix",
      "git add --force"
    ],
```
Add that to your package.json and let's see if it works

Reply:
This is done, it works locally. I have noticed that `sass-lint-fix` does not fix all warnings

Reply:
How do you mean?

Reply:
The ones that don't get fixed by sass-lint-fix, does husky allow them to be committed. Is hound flagging the ones that are not resolved?

Reply:
They are flagged as warnings and nor errors, that is why husky will let it pass